manual/AddingWikis.md

```diff
@@ -63,7 +63,7 @@ eng
 	config
 		logo.png
 		favicon.ico
-		setup.php
+		preLocalSettings.php
 		CustomSettings.php
```

Comment:
Does `CustomSettings.php` still exist? This PR includes a renamed file: `wiki-init/config/CustomSettings.php → ...te/wiki-init/config/postLocalSettings.php` which makes me think this should be updated.